moveit_core/collision_detection/include/moveit/collision_detection/collision_tools.h

```diff
@@ -48,6 +48,7 @@
 
 namespace collision_detection
 {
+// Perfectly "pure" helper functions
```

Comment:
why state this here? It doesn't document anything noteworthy about this free function.